openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -842,11 +842,10 @@ Here is another example JWT not only proving possession of a private key but als
 
 When a W3C Verifiable Presentation as defined by [@VC_DATA_2.0] or [@VC_DATA] signed using Data Integrity is used as Key Proof, it MUST contain the following elements:
 
-  * `holder`: OPTIONAL. MUST be equivalent to the controller identifier (e.g. DID) for the `verificationMethod` value identified by the `proof.verificationMethod` property.
-
```

Comment:
I think the empty line is causing the rendering error but I fixed the indention as well